frontend/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "https://gandalf-backend.herokuapp.com",
+    "target": "https://gandalf-frontend.herokuapp.com",
```

Comment:
are you sure you need this change?

Reply:
![image](https://user-images.githubusercontent.com/60847370/99361352-030f4c00-28c3-11eb-9d9f-a8f5e27cbd8f.png)
I have taken this url from our chat. i had to restore it after change oh local server